README.md

```diff
@@ -1,10 +1,23 @@
 # Dockerized repository
 
 ## Configuration
+
+### Templated configuration
+
 Configure variables in .env. See [.env.example](.env.example) for template.
 Reasonable defaults are provided.
+We provide detailed documentation for [variable-expansion](https://doc.linkurious.com/admin-manual/latest/configure/#variable-expansion)
+
 You will most likely need to set your private repository host, if images are not available locally.
 
+This mode has limitations and will be dropped in the future.
+
+### Configuration overlay
+
+As an alternative, you may put a partial configuration file in your data folder: `data/config/production-overlay.json`.
+This file will be overlayed on top of your current production.json file. If no `production.json` file is provided, defaults will be generated.
```

Comment:
maybe explain what this means:
- having these values in the config: the config value is ignored
- changing the config value (via config file or ui): the change is ignored, the value from the overlay continues to be used
- be careful about possibly creating inconsistent config: the merged result of config + overlay might not pass validation

Reply:
pushed an edit to match your proposals.